monai/bundle/reference_resolver.py

```diff
@@ -202,6 +202,21 @@ def normalize_id(cls, id: str | int) -> str:
         """
         return str(id).replace("#", cls.sep)  # backward compatibility `#` is the old separator
 
+    @classmethod
+    def normalize_meta_id(cls, config: Any) -> Any:
+        """
+        Update deprecated id use `DEPRECATED_ID_MAPPING`.
+
+        Args:
+            config: input config to be updated.
+        """
```

Comment:
```suggestion
    def normalize_meta_id(self, config: Any) -> Any:
        """
        Update deprecated identifiers in `config` using `DEPRECATED_ID_MAPPING`. This will replace names that are marked as deprecated with their replacement.

        Args:
            config: input config to be updated.
        """
```